index.html

```diff
@@ -713,79 +713,59 @@ <h2>Patrocínio</h2>
 
                 <li>
-                  <a data-toggle="collapse" class="collapsed" href="#faq1">Non consectetur a erat nam at lectus urna duis? <i class="fa fa-minus-circle"></i></a>
+                  <a data-toggle="collapse" class="collapsed" href="#faq1">Quem pode participar? <i class="fa fa-minus-circle plus"></i></a>
                   <div id="faq1" class="collapse" data-parent="#faq-list">
                     <p>
-                      Feugiat pretium nibh ipsum consequat. Tempus iaculis urna id volutpat lacus laoreet non curabitur gravida. Venenatis lectus magna fringilla urna porttitor rhoncus dolor purus non.
+                      Qualquer pessoa que tenha interesse em colaborar com projetos de código aberto, independente da área de atuação ou nível de conhecimento técnico.
                     </p>
                   </div>
                 </li>
 
                 <li>
-                  <a data-toggle="collapse" href="#faq2" class="collapsed">Feugiat scelerisque varius morbi enim nunc faucibus a pellentesque? <i class="fa fa-minus-circle"></i></a>
+                  <a data-toggle="collapse" href="#faq2" class="collapsed">É preciso saber programar? <i class="fa fa-minus-circle plus"></i></a>
                   <div id="faq2" class="collapse" data-parent="#faq-list">
                     <p>
-                      Dolor sit amet consectetur adipiscing elit pellentesque habitant morbi. Id interdum velit laoreet id donec ultrices. Fringilla phasellus faucibus scelerisque eleifend donec pretium. Est pellentesque elit ullamcorper dignissim. Mauris ultrices eros in cursus turpis massa tincidunt dui.
+                      Não. Há diversas formas de contribuir, inclusive com design, documentação e traduções.
                     </p>
                   </div>
                 </li>
 
                 <li>
-                  <a data-toggle="collapse" href="#faq3" class="collapsed">Dolor sit amet consectetur adipiscing elit pellentesque habitant morbi? <i class="fa fa-minus-circle"></i></a>
+                  <a data-toggle="collapse" href="#faq3" class="collapsed">Se participar eu ganho camisetas? <i class="fa fa-minus-circle plus"></i></a>
                   <div id="faq3" class="collapse" data-parent="#faq-list">
                     <p>
-                      Eleifend mi in nulla posuere sollicitudin aliquam ultrices sagittis orci. Faucibus pulvinar elementum integer enim. Sem nulla pharetra diam sit amet nisl suscipit. Rutrum tellus pellentesque eu tincidunt. Lectus urna duis convallis convallis tellus. Urna molestie at elementum eu facilisis sed odio morbi quis
+                      As camisetas são prêmios da Digital Ocean e Dev para os primeiros 50.000 a abrir 4 pull requests durante o período da Hacktoberfest - de 1º a 31 de Outubro. Veja mais detalhes em <a target="_blank" href="https://hacktoberfest.digitalocean.com"><i class="fa fa-external-link" aria-hidden="true"></i> hacktoberfest.digitalocean.com</a>.
```

Comment:
Please, break lines at 80th column, more or less, to make it more readable in the code.

Reply:
Also, check my previous comment with qa suggestions about _what do I earn?_. I think that can come just before this one.